hyrax.gemspec

```diff
@@ -26,6 +26,12 @@ EOF
   spec.version       = Hyrax::VERSION
   spec.license       = 'Apache2'
 
+  # Be pessimistic, rails does not follow sem-ver conventions and
+  # minor version releases can include breaking changes; see
+  # http://guides.rubyonrails.org/maintenance_policy.html
+  # https://github.com/projecthydra-labs/hyrax/issues/1028
+  spec.add_dependency 'rails', '>= 5.0.0', '< 5.2'
```

Comment:
Note that `rails plugin` itself creates a fairly pessimistic constraint, like:
- `s.add_dependency "rails", "~> 5.1.1"`